settings.py

```diff
@@ -3,7 +3,25 @@
 
 logger = logging.getLogger(__name__)
 
-LOGGING_FORMAT = "%(asctime)s|%(levelname)s: sdx-sequence: %(message)s"
+LOGGING_FORMAT = "%(message)s"
```

Comment:
Question answered - we think it's possible for Splunk to deal with both, so this is totally fine and I'll put something on the cards to get the config updated 👍